api/server.js

```diff
@@ -0,0 +1,217 @@
+const express = require("express");
+const morgan = require("morgan");
+const helmet = require("helmet");
+const cors = require("cors");
+const userDb = require("../data/helpers/userDb");
+const postsDb = require("../data/helpers/postDb");
+const tagDb = require("../data/helpers/tagDb");
+
+const server = express();
+
+// MIDDLEWARE
+
+// Applying Middleware
+server.use(morgan("short"));
+server.use(helmet());
+server.use(cors());
+server.use(express.json());
+
+// Custom Middleware
+function uppercaseUsername(req, res, next) {
+  let { name } = req.body;
+
+  if (!name) {
+    return res
```

Comment:
Great job trouble shooting and figuring out this `return` issue. I saw Justin spreading your knowledge in the help channel, that was a really good thing to share in stand-up.